src/Hydration.js

```diff
@@ -0,0 +1,42 @@
+const moment = require('moment');
+moment().format(); 
+
+class Hydration {
+  constructor(id) {
+    this.id = id;
+  }
+
+  getDataFromId(data) {
+      this.data = data.filter(instance => instance.userID === this.id);
+  }
+
+  getAverageFlOzPerDay() {
+    let totalOz = this.data.reduce((currentTotalOz, currentDay) => {
+      return currentTotalOz + currentDay.numOunces;
+    }, 0);
+    return totalOz / this.data.length;
+  }
+  
+  getFlOzOnDate(date) {
+    let hydrationOnDate = this.data.find((_day, index) => {
+      return date === this.data[index].date;
+    });
+    return hydrationOnDate.numOunces;
+  }
+
+  getOzForPreviousSevenDays(day) {
+    let currentDay = moment(day, 'YYYY/MM/DD').add(1, 'day');
+    let weekAgo = currentDay.clone().subtract(8,'d');
+    let daysInWeek = this.data.filter(instance => {
+      let dayInLoop = moment(instance.date, 'YYYY/MM/DD').isBetween(weekAgo, currentDay);
+      return dayInLoop
```

Comment:
We might want to rename some of these variables for readability, but functionality is working as expected! Merging now.